eduquest/app/Register/TeacherRegistration/page.tsx

```diff
@@ -0,0 +1,55 @@
+import Image from "next/image";
+import Link from "next/link";
+
+export default function TeacherRegistration() {
+  return (
+    <main className="px-4 bg-gradient-to-b from-amber-100 to-amber-500 min-h-screen flex items-center flex-col">
```

Comment:
We'll need to create form components, since this is getting a bit messy to read.